src/selectors.ts

```diff
@@ -0,0 +1,28 @@
+import { createSelector } from 'reselect';
+import { Map } from 'immutable';
+import { TezosFilter } from 'conseiljs';
+import {
+  DataView,
+  ArronaxImmutableState,
+  ArronaxState,
+  ImmutableMap
+} from './types';
+
+export const selectFiltersDomain = (
+  state: ArronaxImmutableState
+): ImmutableMap<TezosFilter> => state.get('filters');
+
+export const makeSelectFilters = createSelector<
```

Comment:
According to the `reselect` tutorial, selectors are named using the convention `xSelector`. How how come you are using a different naming convention?

Reply:
previously that was the most common naming convention, should I change it ? ;-)